permits/views.py

```diff
@@ -708,7 +750,6 @@ def permit_request_actors(request, permit_request_id):
         permit_request=permit_request,
         current_step_type=models.StepType.ACTORS,
     )
-    requires_payment = services.permit_requests_has_paid_wot(permit_request)
```

Comment:
TODO: Check, this should be on the main branch already, YC-131, should not be removed